files/default/sql/ddl/0.9.0__initial_tables.sql

```diff
@@ -617,6 +617,7 @@ CREATE TABLE `jupyter_project` (
   `pid` bigint(20) NOT NULL,
   `project_id` int(11) NOT NULL,
   PRIMARY KEY (`port`),
+  UNIQUE KEY `unique_hdfs_user` (`hdfs_user_id`),
   KEY `hdfs_user_idx` (`hdfs_user_id`),
```

Comment:
We should condense the `unique key` and the `key` in the `hdfs_user_id`. There is no point having both

Reply:
from what I understand key and unique key are different. key creates a DB index while unique key is a constraint.

Reply:
fixed

Reply:
You cannot assume the name of the constraint. You have to query the information schema to get the correct name by querying the `information_schema` db